src/java/org/apache/nutch/segment/SegmentMerger.java

```diff
@@ -120,20 +121,18 @@
  * 
  * @author Andrzej Bialecki
  */
-public class SegmentMerger extends Configured implements Tool,
-    Mapper<Text, MetaWrapper, Text, MetaWrapper>,
-    Reducer<Text, MetaWrapper, Text, MetaWrapper> {
+public class SegmentMerger extends Configured implements Tool{
   private static final Logger LOG = LoggerFactory
       .getLogger(MethodHandles.lookup().lookupClass());
 
   private static final String SEGMENT_PART_KEY = "part";
   private static final String SEGMENT_SLICE_KEY = "slice";
 
-  private URLFilters filters = null;
-  private URLNormalizers normalizers = null;
-  private SegmentMergeFilters mergeFilters = null;
-  private long sliceSize = -1;
-  private long curCount = 0;
+  private static URLFilters filters = null;
+  private static URLNormalizers normalizers = null;
+  private static SegmentMergeFilters mergeFilters = null;
+  private static long sliceSize = -1;
+  private static long curCount = 0;
```

Comment:
See comments in Generator etc. regarding static variables shared between mapper and reducer classes.